app/webpack/javascripts/modules/external_users/claims/fee_calculator/FeeCalculator.GraduatedPrice.js

```diff
@@ -156,7 +156,7 @@
 
     responseErrored: function (response) {
       return Boolean(
-        response.hasOwnProperty('responseJSON') &&
+        Object.prototype.hasOwnProperty.call(response, 'responseJSON') &&
```

Comment:
can't say i understand the differences here but has this been checked that it breaks nothing. The cucumber tests do test some fee calc functionality but can't say I would be confident this particular code would be covered by them

Reply:
> can't say i understand the differences here but has this been checked that it breaks nothing. The cucumber tests do test some fee calc functionality but can't say I would be confident this particular code would be covered by them

In short, it prevents call object.prototype methods directly from an object, see: https://eslint.org/docs/rules/no-prototype-builtins.

This has been tested locally and will be tested once deployed to an env.